### PR TITLE
Light doc pass.

### PR DESCRIPTION
**Pull request by vmagelo**
Changes just to docstrings, except to conf.py to get local builds to behave nicely in terms of linking.

In general:
1. changed "user" terminology to "you" to make it more personal
2. where "we" was used, either named actor or went with passive form of sentence 
3. checked for proper user of brand names.

### CODE REVIEW

**Comment by imatiach-msft** [comment]
@vmagelo it looks like a bunch of builds are failing, are you able to see the build output and figure out what changes are causing the failures/fix them?

File: python/interpret_community/explanation/explanation.py [hunk 1]
```diff
@@ -27,12 +27,12 @@ class BaseExplanation(ChainedIdentity):
 
     """The common explanation returned by explainers.
 
-    :param method: The explanation method used to explain the model (e.g. SHAP, LIME).
+    :param method: The explanation method used to explain the model (e.g., SHAP, LIME).
     :type method: str
-    :param model_task: The task of the original model i.e. classification or regression.
+    :param model_task: The task of the original model i.e., classification or regression.
```
Comment (imatiach-msft) [inline python/interpret_community/explanation/explanation.py]: this comma after i.e. and e.g. is really bothering me hehe :), I never do that.

**Comment by vmagelo** [comment]
Will take a look.

File: python/interpret_community/explanation/explanation.py (hunk 1 above)
Comment (vmagelo) [reply python/interpret_community/explanation/explanation.py]: If it's really bothering you, I can go either way on it. :-)  I'm trying to make your life easier.

**Comment by vmagelo** [comment]
@imatiach-msft - made transformations parameter in explainers consistent with interpret-community package as well as with SDK version.

**Comment by imatiach-msft** [comment]
@vmagelo looks great, can I merge?

**Comment by vmagelo** [comment]
@imatiach-msft 

Yes, let's merge. Thanks.

**Comment by imatiach-msft** [comment]
@vmagelo done, merged doc updates

**Pull request merged**